x-pack/test_serverless/functional/test_suites/observability/config.telemetry.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { resolve } from 'path';
+import type { GenericFtrProviderContext } from '@kbn/test';
+import {
+  KibanaEBTUIProvider,
+  KibanaEBTServerProvider,
+} from '@kbn/test-suites-src/analytics/services/kibana_ebt';
+import { services as inheritedServices } from '../../services';
+import { pageObjects } from '../../page_objects';
+import { createTestConfig } from '../../config.base';
+
+type ObservabilityTelemetryServices = typeof inheritedServices & {
+  kibana_ebt_server: typeof KibanaEBTServerProvider;
```

Comment:
I don't see the use of `kibana_ebt_server` service in tests. Do you plan to add more tests later that will call it?

Reply:
Nope, no plans for now so I think it makes sense to drop it. Updated here: https://github.com/elastic/kibana/pull/201310/commits/0585a256345d29b80038f2f2f7b78ffd08ba055b.